pyproject.toml

```diff
@@ -140,14 +140,14 @@ inputs = "*"
 Jinja2 = "*"
 lru-dict = "*"
 matplotlib = "*"
-metadrive-simulator = { version = "0.4.2.3", markers = "platform_machine != 'aarch64'" } # no linux/aarch64 wheels for certain dependencies
+metadrive-simulator-py3-12 = { version = "0.3.0", markers = "platform_machine != 'aarch64'" } # no linux/aarch64 wheels for certain dependencies
```

Comment:
I pushed your branch to our fork. can you install from git? https://github.com/commaai/metadrive/tree/python3.12

Reply:
you'll want to update that branch, 0.2.0 had missing assets. 0.3.0 adds them

Reply:
done

Reply:
PR now uses the git that you linked. I should mention that the repo as it stands does rely on a release to exist in order to download assets for metadrive use (assuming they're not there locally)

It'll work fine since it will download the assets from the release coming from my repo: https://github.com/BBBmau/metadrive/releases/tag/0.3.0 but i figured I'd bring this up. Not sure if you want to leave as is, this is technically a temporary fix until they support it on theirs.